solidstart-tanstackquery-tailwind-modules/src/services/get-org-repos.ts

```diff
@@ -0,0 +1,33 @@
+import FetchApi from './api';
+import { useAuth } from '../auth';
+import { GITHUB_GRAPHQL } from '../utils/constants';
+import { ORGANIZATION_REPOS_QUERY } from './queries/org-repos';
+import { OrgRepoInfo } from '~/types/org-repos';
+
+type Variables = Record<string, string | number | null> | null;
+type Response = {
+  data: OrgRepoInfo;
+};
+
+const getOrgRepos = async (variables: Variables) => {
+  const { authStore } = useAuth();
+
+  const data = {
```

Comment:
Please add type to this data

Reply:
We don't need to do this since the type has been declared in the useAuth(), it carried the types with it.

You can see the image below;

<img width="511" alt="image" src="https://user-images.githubusercontent.com/36667083/217041435-16a093d3-1dfa-44c8-9d1c-4b2ca3aebf4f.png">